pyomo/contrib/doe/doe.py

```diff
@@ -632,7 +636,13 @@ def block_build(b, s):
             for par in self.param:
                 cuid = pyo.ComponentUID(par)
                 var = cuid.find_component_on(b)
```

Comment:
There is another copy of this code starting on line 612. Does it need to be updated also?

Reply:
Yes, they are updated now